content/en/database_monitoring/guide/aurora_autodiscovery.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Configuring Database Monitoring for Amazon Aurora DB Clusters
```

Comment:
@ceciliawatt I agree with your sentiments around avoiding too much focus on autodiscovery. Autodiscovery is the how, not the what. Thoughts on this title? I took it straight from the guide link.

Reply:
That looks good to me